storage/src/main/java/tech/pegasys/teku/storage/server/kvstore/UnblindedBlockKvStoreDatabase.java

```diff
@@ -113,6 +113,12 @@ public Optional<Bytes32> getFinalizedBlockRootBySlot(final UInt64 slot) {
     return dao.getFinalizedBlockAtSlot(slot).map(SignedBeaconBlock::getRoot);
   }
 
+  @Override
+  public Optional<ExecutionPayload> getExecutionPayload(
+      final Bytes32 blockRoot, final UInt64 slot) {
+    return Optional.empty();
```

Comment:
Should this load the block and get the payload from there? Generally wouldn't be used I guess but might be worth implementing defensively just to avoid surprises.

Reply:
we can't switch back to unblinded, i could fetch the block and then get it off of that potentially?

Reply:
Yeah the unblinded database would get a payload by loading the full block, then getting the payload from it. Bit wasteful but we're really only implementing it to avoid surprises.